src/pkg/packaging/dir.proj

```diff
@@ -172,8 +172,6 @@
 
     <ItemGroup>
       <!-- The list of packages we are servicing -->
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\Microsoft.DotNet.PlatformAbstractions.csproj" />
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.Extensions.DependencyModel\Microsoft.Extensions.DependencyModel.csproj" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(BuildAllPackages)' == 'true'">
```

Comment:
I think you should add the above list to this item group.

Reply:
Oh , you're right. Thanks. Just fixed it.